### DISCUSSION TITLE
Version update is too fast

### DISCUSSION THREAD

**Discussion by WarlockFish** [casdoor/casdoor#1206] [Ideas]
i like casdoor ,I suggest that there is a planned release version.
Now the update is too fast. One version is updated for each problem. The change log cannot be displayed directly 。
 Docker is also updated quickly and takes up a lot of space. The gap between versions is too small. It is very difficult to select a version when i install the casdoor.

**Comment by hsluoyz** [comment]
Using the latest version is recommended.
